src/compositedataframe.jl

```diff
@@ -138,7 +138,10 @@ DataFrames.index(cdf::AbstractCompositeDataFrame) = DataFrames.Index(names(cdf))
 #########################################
 
 Base.getindex(cdf::AbstractCompositeDataFrame, col_inds::DataFrames.ColumnIndex) = getfield(cdf, col_inds)
-Base.getindex{T <: DataFrames.ColumnIndex}(cdf::AbstractCompositeDataFrame, col_inds::AbstractVector{T}) = CompositeDataFrame(Any[ getfield(cdf, col_inds[i]) for i = 1:length(col_inds) ], names(cdf)[col_inds])
+function Base.getindex(cdf::AbstractCompositeDataFrame, col_inds::AbstractVector{T}
+                      ) where {T<:DataFrames.ColumnIndex}
```

Comment:
I'd keep this on the previous line, I don't think it's common to start a line with `)`.